### PR TITLE
new revolver, price change

### PR DESCRIPTION
**Pull request by Jackal298**
New cargo order, Deckard and Mateba revolver prices

## About the PR

This project grew out of my previous one, I wanted to be able to order a revolver in cargo, since all the existing revolvers did not fit due to the fact that they were already assigned to someone in the game, I decided to make a new revolver. The Rex revolver is identical in characteristics to the inspector revolver. I also rebalanced the price of Deckard and Mateba's revolver, Deckard costs 7000 credits (now it matches its description), Mateba's price is 2500 credits

If you order a revolver in cargo you will receive:
One revolver Rex
Three speedloaders with live ammunition
one box of normal combat 45. magnum

**Media**

Rex 1

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: added a new revolver Rex, it can be purchased in cargo
- tweak: Deckard and Mateb revolver price changed

### CODE REVIEW

**Comment by keronshb** [comment]
There is a current freeze on new guns: https://github.com/space-wizards/space-station-14/issues/8524

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml [hunk 1]
```diff
@@ -65,6 +65,8 @@
     steps: 4
     zeroVisible: true
   - type: Appearance
+  - type: StaticPrice
+    price: 10000
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml:69]: ?

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
There's currently a freeze on new weapons as we culled a lot of the excess ones and even now we still have a few excess. Revolvers in particular aren't balanced amongst themselves already so a new one is just going to be a reskin. The crate change is good.

**Comment by Jackal298** [comment]
> There's currently a freeze on new weapons as we culled a lot of the excess ones and even now we still have a few excess. Revolvers in particular aren't balanced amongst themselves already so a new one is just going to be a reskin. The crate change is good.

I understand you, this PR is doing, I did it with a margin for the future, tomorrow I will do pr to increase damage for .45 magnum and change the smg vector caliber, I wanted to make it possible to buy a revolver in cargo, but due to the fact that the game does not I had a suitable revolver, I decided to make my own, I can try to make the differences between revolvers from each other by altering the delay between shots and shooting modes, in the game the revolvers are very weak and I am trying to improve them so that they can be used

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml (hunk 1 above)
Comment (Jackal298) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml:69]: > ?

I have set the price for the decartes pistol based on its description and rarity, I can lower the price to 7000 credits

**Pull request closed without merging**